### DISCUSSION TITLE
EARS data

### DISCUSSION THREAD

**Discussion by Hannah-hue102** [now-i-know-my-abcd/docs#99] [Stats & Code Help] [answered]
Two questions about EARS (Effortless Assessment Research System). 1) is future EARS data going to be collected? and 2) is the current EARS data technically the 4-year follow up even though it is listed in the 5-year protocol? I read a study that said, "data collected between September 2019 through January 2021, containing the first half of Year 4 follow-up data (total n = 4754; a subset, n = 1,463, had usable ABCD-EARS data)." (Alexander et al., 2024).

**Comment by angielaird** [comment, marked as answer]
1) Yes, ABCD is collecting EARS every year, starting at the Year 4 follow-up (as in the data from Alexander et al., 2024). Android data are available at every follow-up; Apple data availability varies, but starting in the Year 7 follow-up, Apple data will become more consistent in availability and format (all due to Apple privacy issues). The (soon-to-be-released) ABCD 6.0 release will contain EARS summary and raw data from Y4, Y5, and Y6.

2) I'm not sure where it's listed as part of the 5-year protocol exclusively, but yes, in current data release 5.1, EARS is available at the Year 4 follow-up.
